django/db/models/functions/math.py

```diff
@@ -0,0 +1,7 @@
+from django.db.models import Transform
+
+
+class Abs(Transform):
+    """Return the absolute value of the expression."""
```

Comment:
I'm not really sure this docstring is necessary as the implementation is obvious.